package.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "crypto": "0.0.3",
```

Comment:
I've also checked the more popular [pvorb/node-md5](https://github.com/pvorb/node-md5). The ones you found are much lighter and also safe options for the job. :+1: